### PR TITLE
Add Python Example to Swerve Visualization Struct

### PR DESCRIPTION
**Pull request by jasondaming**
@virtuald can you correct this example.  I am not sure how exactly to use this struct.  Hopefully I got this close?

### CODE REVIEW

File: source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst [hunk 1]
```diff
@@ -236,6 +236,27 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
         }
       };
 
+   .. code-block:: python
+
+      from wpiutil import wpistruct
+      import ntcore
+
+      class MyStruct:
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst:244]: This isn't part of this example

File: source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst [hunk 2]
```diff
@@ -236,6 +236,27 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
         }
       };
 
+   .. code-block:: python
+
+      from wpiutil import wpistruct
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst:241]: This import isn't required for this example

File: source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst [hunk 3]
```diff
@@ -236,6 +236,27 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
...
+      class MyStruct:
+        x: int
+        y: bool
+        z: wpistruct.double
+
+      # get the default instance of NetworkTables
+      nt = ntcore.NetworkTableInstance.getDefault()
+
+      # Start publishing an array of module states with the "/SwerveStates" key
+      topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState.struct)
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst:253]: `.struct` isn't needed, should be `topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState)`

File: source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst [hunk 4]
```diff
@@ -236,6 +236,27 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
...
+      self.pub = topic.publish()
+
+      def periodic(self):
+        # Periodically send a set of module states
+        self.pub.set(MyStruct(frontLeftState,frontRightState,backLeftState,backRightState))
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst:258]: `SwerveModuleState`, not `MyStruct`

**Review by virtuald — CHANGES_REQUESTED** [review]
For future struct topic, refer to the tests at https://github.com/robotpy/mostrobotpy/blob/main/subprojects/pyntcore/tests/test_struct_topic.py

File: source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst [hunk 5]
```diff
@@ -236,6 +236,22 @@ By recording a set of swerve module states using :ref:`NetworkTables <docs/softw
...
+      # get the default instance of NetworkTables
+      nt = ntcore.NetworkTableInstance.getDefault()
+
+      # Start publishing an array of module states with the "/SwerveStates" key
+      topic = nt.getStructArrayTopic("/SwerveStates", SwerveModuleState)
+      self.pub = topic.publish()
+
+      def periodic(self):
+        # Periodically send a set of module states
+        self.pub.set(SwerveModuleState(frontLeftState,frontRightState,backLeftState,backRightState))
```
Comment (virtuald) [inline source/docs/software/kinematics-and-odometry/swerve-drive-kinematics.rst:253]: Oh, you're right, this should be `self.pub.set([frontLeftState,frontRightState,backLeftState,backRightState])`

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**